index.js

```diff
@@ -5,7 +5,7 @@ const pAny = require('p-any');
 const pTimeout = require('p-timeout');
 
 const appleCheck = async options => {
-	const {body} = await got('http://captive.apple.com/hotspot-detect.html', {
+	const {body} = await got(options.url, {
```

Comment:
You cannot just change the URL here. Look at the surrounding code. (https://github.com/sindresorhus/is-online/pull/67/files#diff-168726dbe96b3ce427e7fedce31bb0bcR15)